desktop/desktop.scss

```diff
@@ -0,0 +1,10 @@
+@media (min-width: 640px) {
```

Comment:
Same comment about re-creating the utility classes

Reply:
this code wasn't being used so I ended up removing it